projects/golang/go/1.16/rpmbuild/SPECS/golang.spec

```
@@ -175,6 +175,7 @@ Patch20:       0020-go-1.16.15-eks-syscall-os-exec-reject-environ.patch
 Patch21:       0021-go-1.16.15-eks-crypto-rand-properly-handle-la.patch
 Patch22:       0022-go-1.16.15-eks-os-exec-return-clear-error-for.patch
 Patch23:       0023-go-1.16.15-eks-path-filepath-do-not-remove-pr.patch
+Patch24:       0024-go-1.16.15-eks-os-net-http-avoid-escapes-from-os.Dir.patch
```

I wonder if this is what's breaking the tests?

```suggestion
Patch24:       0024-go-1.16.15-eks-os-net-http-avoid-escapes-from-os.patch
```